### PR TITLE
Update documentation to refect `strawberry-django`

### PR DESCRIPTION
**Pull request by baseplate-admin**
See discussion on : https://github.com/strawberry-graphql/strawberry.rocks/issues/474#issuecomment-1424438765

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

* https://github.com/strawberry-graphql/strawberry.rocks/issues/474

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/integrations/django.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ urlpatterns = [
 ]
 ```
 
+Strawberry core only provides a GraphQL view for Django, Strawberry GraphQL Django provides integration with the models.
```
Comment (DoctorJohn) [inline docs/integrations/django.md:23]:
```suggestion
Strawberry only provides a GraphQL view for Django, [Strawberry GraphQL Django](https://github.com/strawberry-graphql/strawberry-graphql-django) provides integration with the models.
```

IMHO a link to strawberry-graphql-django would be useful :)

nitpick, feel free to ignore: we might want to drop "core" from the wording. I know Patrick used it but technically the core library is just called Strawberry (or Strawberry GraphQL), could be confusing to some to suddenly read "core".

**Review by DoctorJohn — APPROVED** [review]
Thanks for the PR @baseplate-admin! I left one optional comment, lemme know what you think.

**Comment by baseplate-admin** [comment]
Thanks for the review. I changed the wordings. Please take another look. Thanks :D

**Review by patrick91 — APPROVED** [review]

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**